pulsar-broker/src/test/java/org/apache/pulsar/broker/service/ExclusiveProducerTest.java

```diff
@@ -331,7 +332,12 @@ public void topicDeleted(String ignored, boolean partitioned) throws Exception {
         p1.send("msg-1");
 
         if (partitioned) {
-            admin.topics().deletePartitionedTopic(topic, true);
+            try {
+                admin.topics().deletePartitionedTopic(topic, true);
+                fail("expected error because partitioned topic has active producer");
+            } catch (PulsarAdminException.ServerSideErrorException e) {
```

Comment:
ServerSideError is not very user friendly.
I would expect 'Conflict' or the same thing that happens when you do not use the 'force' option

Reply:
@eolivelli I return the same TopicBusyException as other cases when topic deletion is not possible. 
If it needs to be translated into some other error for pulsar admin/rest API I suggest we create a bug and resolve this in a different PR 

https://github.com/apache/pulsar/blob/34c2262d16591fe9af6ce88ad58c72e3c017d4f1/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java#L1129-L1148